linksharing/handler.go

```diff
@@ -35,13 +39,29 @@ type HandlerConfig struct {
 
 	// Templates location with html templates.
 	Templates string
+
+	// TxtRecordTTL is the duration for which an entry in the txtRecordCache is valid.
+	TxtRecordTTL time.Duration
+}
+
+type txtRecord struct {
+	access    string
```

Comment:
might save some CPU time if you stored this parsed, like *uplink.Access instead of string

Reply:
done